src/components/MoneyRequestAmountInput.tsx

```diff
@@ -0,0 +1,206 @@
+import React, {useCallback, useEffect, useRef, useState} from 'react';
```

Comment:
For now I copied and pasted code from `MoneyRequestAmountForm`, we're working on making the underlying amount input reusable here https://github.com/Expensify/App/issues/40382